### PR TITLE
update: typo for greetings.yml workflow script

### PR DESCRIPTION
**Pull request by 0xabdulkhalid**
## Fixes Issue
- #871 

## Changes proposed
- Some minor changes in typo of `greetings.yml` workflow script
- Changed typo for both `issue` & `pull request`

### Screenshot:
| | |
| :---: | :---: |
| **CURRENT** | | 
| **PROPOSED CHANGES** |  | 

## Note to reviewers
- The required necessary labels like `gssoc23` and `level` labels are not added for the merged PR: #870 (The issue is not closed to).
- I'm note this here because the `GSSoC`'s Backend can't count this PR without necessary labels.
- Please make sure to add those necessary labels, Thanks!

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -17,5 +17,5 @@ jobs:
       - uses: EddieHubCommunity/gh-action-community/src/welcome@main
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          issue-message: "Hello ${{ github.actor }}, thanks for opening a issue, your contribution is valuable to us. The maintainers will review this issue and provide feedback as soon as possible."
-          pr-message: "Hello ${{ github.actor }}, thanks for rising a Pull request, your contribution is valuable to us. The maintainers will review this Pull Request and provide feedback as soon as possible. Keep the great work up!"
+          issue-message: "Hello @${author}! \n Thank you for raising this issue, your contribution is valuable to us. \n\nPlease make sure to follow our [Contributing Guidelines. 💪🏻](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) \n\nOur review team will carefully assess the issue and reach out to you soon! 😇 \n We appreciate your patience! "
```
Comment (CBID2) [inline .github/workflows/greetings.yml:20]: [Proofreading]
```suggestion
          issue-message: "Hello @${author}! \n Thank you for raising this issue! 😊 Your contribution is valuable to us! 😊 \n\nPlease make sure to follow our [Contributing Guidelines. 💪🏻](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) \n\nOur review team will carefully assess the issue and reach out to you soon! 😇 \n We appreciate your patience! "
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just made a few tweaks @0xabdulkhali! :)

**Comment by 0xabdulkhalid** [comment]
The typo has been updated as per your request @CBID2 !

**Review by rupali-codes — APPROVED** [review]

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -17,5 +17,5 @@ jobs:
...
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          issue-message: "Hello ${{ github.actor }}, thanks for opening a issue, your contribution is valuable to us. The maintainers will review this issue and provide feedback as soon as possible."
-          pr-message: "Hello ${{ github.actor }}, thanks for rising a Pull request, your contribution is valuable to us. The maintainers will review this Pull Request and provide feedback as soon as possible. Keep the great work up!"
+          issue-message: "Hello @${author}! \n Thank you for raising this issue! 😊 Your contribution is valuable to us! 😊 \n\nPlease make sure to follow our [Contributing Guidelines. 💪🏻](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) \n\nOur review team will carefully assess the issue and reach out to you soon! 😇 \n We appreciate your patience! "
+          pr-message: "Thank you, @${author}, for creating this pull request and contributing to LinksHub! 💗\n\n The maintainers will review this Pull Request and provide feedback as soon as possible! 😇\nWe appreciate your patience and contribution, Keep the great work up! 😀"
```
Comment (CBID2) [inline .github/workflows/greetings.yml:21]: [Proofreading]
```suggestion
          pr-message: "Thank you, @${author}, for creating this pull request and contributing to LinksHub! 💗\n\n The maintainers will review this Pull Request and provide feedback as soon as possible! 😇\nWe appreciate your patience and contribution, Keep up the great work! 😀"
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just made another tweak @0xabdulkhalid.

**Comment by 0xabdulkhalid** [comment]
Done @CBID2 !

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**